[cuDNN] build failure on x64-linux

**Host Environment**
- Linux
- Compiler: GCC 12.1

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install cudnn`

**Failure logs**
```
Building cuda[core]:x64-linux...
-- Installing port from location: /data/devel/vcpkg/buildtrees/versioning_/versions/cuda/4e8939c3547a570a73098c091da7b9e28f71015c
-- Found CUDA V11.6.124
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: /home/umar/.cache/vcpkg/archives/82/82dc4bcaebd0801a3fd661c3d3aef54234f2b3dc9fb9a59d38183432d4360adf.zip
Elapsed time to handle cuda:x64-linux: 45.39 ms
Installing 5/36 cudnn:x64-linux...
Building cudnn[core]:x64-linux...
-- Installing port from location: /data/devel/vcpkg/buildtrees/versioning_/versions/cudnn/62f2616b19fe2b3b7dbc8a81e190900819f2ba3e
-- Note: cudnn only supports dynamic library linkage. Building dynamic library.
-- Found CUDA V11.6.124
-- CUDNN_INCLUDE_DIR: /usr/include
-- CUDNN_LIBRARY: CUDNN_LIBRARY-NOTFOUND
CMake Error at buildtrees/versioning_/versions/cudnn/62f2616b19fe2b3b7dbc8a81e190900819f2ba3e/portfile.cmake:71 (message):
  Please install CUDNN using your system package manager (the same way you
  installed CUDA).  For example: apt install libcudnn8-dev.
Call Stack (most recent call first):
  scripts/ports.cmake:146 (include)


error: building cudnn:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+cudnn
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[cudnn]+Build+error
Include '[cudnn] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
    vcpkg-tool version: 2022-05-05-67e17c1782801cf481be9ac0b3765dff3e4bdeb8
    vcpkg-scripts version: 14e7bb4ae 2022-05-10 (27 hours ago)


-- Running vcpkg install - failed
CMake Error at /data/devel/vcpkg/scripts/buildsystems/vcpkg.cmake:858 (message):
  vcpkg install failed.  See logs for more information:
  /data/devel/arrayfire/build_vcpkg/vcpkg-manifest-install.log
Call Stack (most recent call first):
  build_vcpkg/CMakeFiles/3.23.1/CMakeSystem.cmake:6 (include)
  CMakeLists.txt:12 (project)


-- Configuring incomplete, errors occurred!
```


**Additional context**
I was able to resolve this by adding `/usr/` in the portfile and FindCUDNN.cmake. I can create a PR if this is acceptable.

--- a/ports/cudnn/FindCUDNN.cmake
+++ b/ports/cudnn/FindCUDNN.cmake
@@ -28,7 +28,7 @@ find_path(CUDNN_INCLUDE_DIR NAMES cudnn.h cudnn_v8.h cudnn_v7.h
   HINTS $ENV{CUDA_PATH} $ENV{CUDA_TOOLKIT_ROOT_DIR} $ENV{cudnn} $ENV{CUDNN} $ENV{CUDNN_ROOT_DIR} /usr/include
   PATH_SUFFIXES cuda/include include)
 find_library(CUDNN_LIBRARY NAMES cudnn cudnn8 cudnn7
-  HINTS $ENV{CUDA_PATH} $ENV{CUDA_TOOLKIT_ROOT_DIR} $ENV{cudnn} $ENV{CUDNN} $ENV{CUDNN_ROOT_DIR} /usr/lib/x86_64-linux-gnu/
+  HINTS $ENV{CUDA_PATH} $ENV{CUDA_TOOLKIT_ROOT_DIR} $ENV{cudnn} $ENV{CUDNN} $ENV{CUDNN_ROOT_DIR} /usr/lib/x86_64-linux-gnu/ /usr/
   PATH_SUFFIXES lib lib64 cuda/lib cuda/lib64 lib/x64 cuda/lib/x64)
 if(EXISTS "${CUDNN_INCLUDE_DIR}/cudnn.h")
   file(READ ${CUDNN_INCLUDE_DIR}/cudnn.h CUDNN_HEADER_CONTENTS)

--- a/ports/cudnn/portfile.cmake
+++ b/ports/cudnn/portfile.cmake
@@ -19,7 +19,7 @@ find_path(CUDNN_INCLUDE_DIR NAMES cudnn.h cudnn_v8.h cudnn_v7.h
   PATH_SUFFIXES cuda/include include)
 message(STATUS "CUDNN_INCLUDE_DIR: ${CUDNN_INCLUDE_DIR}")
 find_library(CUDNN_LIBRARY NAMES cudnn cudnn8 cudnn7
-  HINTS ${CUDA_TOOLKIT_ROOT} $ENV{CUDA_PATH} $ENV{CUDA_TOOLKIT_ROOT_DIR} $ENV{cudnn} $ENV{CUDNN} $ENV{CUDNN_ROOT_DIR} /usr/lib/x86_64-linux-gnu/
+  HINTS ${CUDA_TOOLKIT_ROOT} $ENV{CUDA_PATH} $ENV{CUDA_TOOLKIT_ROOT_DIR} $ENV{cudnn} $ENV{CUDNN} $ENV{CUDNN_ROOT_DIR} /usr/lib/x86_64-linux-gnu/ /usr/
   PATH_SUFFIXES lib lib64 cuda/lib cuda/lib64 lib/x64 cuda/lib/x64)
 message(STATUS "CUDNN_LIBRARY: ${CUDNN_LIBRARY}")
 if(EXISTS "${CUDNN_INCLUDE_DIR}/cudnn.h")

--- a/ports/cudnn/vcpkg.json
+++ b/ports/cudnn/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "cudnn",
   "version": "7.6.5",
-  "port-version": 5,
+  "port-version": 6,
   "description": "NVIDIA's cuDNN deep neural network acceleration library.",
   "homepage": "https://developer.nvidia.com/cudnn",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1734,7 +1734,7 @@
     },
     "cudnn": {
       "baseline": "7.6.5",
-      "port-version": 5
+      "port-version": 6
     },
     "cunit": {
       "baseline": "2.1.3",

--- a/versions/c-/cudnn.json
+++ b/versions/c-/cudnn.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "aa008ef1b1c958146532dbbdabd76b62f9558653",
+      "version": "7.6.5",
+      "port-version": 6
+    },
+    {
       "git-tree": "62f2616b19fe2b3b7dbc8a81e190900819f2ba3e",
       "version": "7.6.5",
       "port-version": 5

